site/content/how-to/traffic-management/snippets-filters.md

```diff
@@ -0,0 +1,353 @@
+---
+title: "Use the SnippetsFilter API"
+weight: 800
+toc: true
+docs: "DOCS-000"
+---
+
+This topic introduces Snippets and how to implement them using the `SnippetsFilter` API. It provides an example of how to use a Snippet for rate limiting, and how to investigate errors caused by misconfiguration.
+
+---
+
+## Overview
+
+Snippets allow users to insert NGINX configuration into different contexts of the
+NGINX configurations that NGINX Gateway Fabric generates.
+
+Snippets are for advanced NGINX users who need more control over the generated NGINX configuration,
+and can be used in cases where Gateway API resources or NGINX extensions policies don't apply.
+
+---
+
+## Disadvantages of Snippets
+
+{{< warning >}} We recommend managing NGINX configuration through Gateway API resources, [first-class policies]({{< relref "overview/custom-policies.md" >}}), and other existing NGINX extensions
```

Comment:
@sjberman is "other existing NGINX extensions" even correct? What would those be, I'm not sure. Are the main ways just Gateway API resources and custom policies?

Reply:
Yeah, our extensions are in the form of policies, so you're right.

Reply:
Thought we have the NginxProxy resource, which isn't technically a policy. And I don't believe is in the policy doc.

Reply:
Is there any resource I can link to for the "Other existing NGINX extensions" because even I don't really know what that's referencing specifically.

Reply:
I guess you could link to the data plane configuration page for global settings.